types/value.go

```diff
@@ -433,14 +431,6 @@ func (v StringValue) LessThan(q Value) bool {
 func (v StringValue) String() string {
 	s := v.V
 
```

Comment:
I have mixed feelings about this change because I'm concerned about unintended consequences. I agree it's probably the right thing to do though.

Reply:
Well there certainly are no immediate consequences. The only other place it's used is in advancedio, where I also assume we don't want extra quotes. [Here](https://github.com/getgort/gort/blob/develop/adapter/advancedio.go#L78) and a few lines below. We should probably figure out the proper behavior for `Value` and actually document it.